### PR TITLE
Adds proposal for benchmark configs

### PR DESCRIPTION
**Pull request by chrisvdg**
Proposal of benchmark configs for beta2
Results of the proposal ran locally can be found here:
https://github.com/chrisvdg/0-stor/tree/1.1.0-beta-2-benchmarked/benchmark/benchmarks_local
https://github.com/chrisvdg/0-stor/tree/1.1.0-beta-2-benchmarked/benchmark/profiling_local

### CODE REVIEW

**Comment by GlenDC** [comment]
Cool, will play with it today, and review it when I did.

**Comment by chrisvdg** [comment]
Keep in mind that with changes from #498 the range values of the benchmarks need encapsulated with `[ ]` to turn them into lists

File: benchmark/benchmarks_local/datashards_vs_metashards/read_bench.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+benchmarks:
```
Comment (zaibon) [inline benchmark/benchmarks_local/datashards_vs_metashards/read_bench.yaml]: Not sure this is usefull.
Cause even if you change the data_shards number, the distribution configuration remain the same.
So you will always hit the same number of zstordb

File: benchmark/benchmarks_local/datashards_vs_metashards/write_bench.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+benchmarks:
```
Comment (zaibon) [inline benchmark/benchmarks_local/datashards_vs_metashards/write_bench.yaml]: Not sure this is usefull.
Cause even if you change the data_shards number, the distribution configuration remain the same.
So you will always hit the same number of zstordb

File: benchmark/benchmarks_local/read_vs_write/bench.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+benchmarks:
+- prime_parameter:
```
Comment (zaibon) [inline benchmark/benchmarks_local/read_vs_write/bench.yaml]: value size is not really something we care about here.
So i would jus keep the second test

File: benchmark/profiling_local/compression/gzip_bench.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
+# Benchmarks snappy compression types
```
Comment (zaibon) [inline benchmark/profiling_local/compression/gzip_bench.yaml]: snappy =>  gzip

File: benchmark/profiling_local/compression/lz4_bench.yaml [hunk 5]
```diff
@@ -0,0 +1,33 @@
+# Benchmarks snappy compression types
```
Comment (zaibon) [inline benchmark/profiling_local/compression/lz4_bench.yaml]: snappy => lz4

File: benchmark/benchmarks_local/datashards_vs_metashards/read_bench.yaml (hunk 1 above)
Comment (GlenDC) [reply benchmark/benchmarks_local/datashards_vs_metashards/read_bench.yaml]: I guess that's a failure in the bench config to not be able to handle this in sync?

Comment (GlenDC) [reply benchmark/benchmarks_local/datashards_vs_metashards/read_bench.yaml]: Nevermind, it seems the orchestrator python (config) code already adapts the number of zstordb servers based on the data+parity shards (if you define (any of) those). So I think just the data+parity shards test is good enough. We can also test replication tests in range, by just defining the data shards. But the config as it is now, isn't useful at all. Will adapt the configs.

File: benchmark/benchmarks_local/blocksize_vs_valsize/read_bench.yaml [hunk 6]
```diff
@@ -0,0 +1,28 @@
+benchmarks:
+- prime_parameter:
+    id: block_size   
+    range: [1048576, 2097152, 4194304, 1048576]
```
[the remark this answers is not in the document]
Comment (GlenDC) [reply benchmark/benchmarks_local/blocksize_vs_valsize/read_bench.yaml]: This can be resolved as part of #507.

**Pull request merged**